### PR TITLE
fix: changing model scaling algorithm

### PR DESCRIPTION
**Pull request by LoneGit**

### CODE REVIEW

File: src/figure_editor.py [hunk 1]
```diff
@@ -180,7 +181,10 @@ def __init__(self, methods, captions):
                 act_pos, act_neg, act_set = methods[col, param]
                 btn_pos.clicked.connect(act_pos)
                 btn_neg.clicked.connect(act_neg)
-                edit.textChanged.connect(act_set)
+                if col == 1:
```
Comment (lvjonok) [inline src/figure_editor.py:184]: what is col == 1 ?

**Review by lvjonok — COMMENTED** [review]
small comments, but okay

File: src/gui_utils.py [hunk 2]
```diff
@@ -219,23 +219,27 @@ def createStlActorInOrigin(filename, colorize=False):
...
-    transform.Translate(s.hardware.plane_center_x, s.hardware.plane_center_y, s.hardware.plane_center_z)
-    transform.Translate(s.slicing.originx, s.slicing.originy, s.slicing.originz)
 
-    transform.RotateZ(s.slicing.rotationz)
-    transform.RotateX(s.slicing.rotationx)
-    transform.RotateY(s.slicing.rotationy)
-
-    transform.Scale(s.slicing.scalex, s.slicing.scaley, s.slicing.scalez)
+    m = vtkMatrix4x4()
+    for i in range(3):
```
Comment (l1va) [inline src/gui_utils.py:235]: 3? please add comments everywhere. 
And maybe better to have separate functions that create matrices from data

**Review by lvjonok — APPROVED** [review]
good

**Review by l1va — APPROVED** [review]

**Pull request merged**